tools/cli/bin/jetpack.js

```diff
@@ -2,24 +2,52 @@
 
 import process from 'process';
 import { fileURLToPath } from 'url';
-import { checkCliLocation, compareComposerVersion } from '../helpers/checkEnvironment.js';
 
 /**
- * Checks for executing the CLI within a different monorepo checkout.
+ * Async import with better error handling.
+ *
+ * @param {string} path - Path to import from.
+ * @returns {*} - Return from `import()`.
  */
-try {
-	await checkCliLocation();
-} catch ( error ) {
-	console.error( error );
-	console.error( 'Something unexpected happened. See error above.' );
-	process.exit( 1 );
+async function guardedImport( path ) {
+	try {
+		return await import( path );
+	} catch ( error ) {
+		const identity = v => v;
+		const chalk = await import( 'chalk' ).then(
+			m => m.default,
+			() => ( {} )
+		);
+
+		if ( error.code === 'ERR_MODULE_NOT_FOUND' ) {
+			// if pnpm install hasn't been run, the import() will fail here.
+			console.error(
+				'Something is missing from your install. Please run `pnpm install` and try again.'
+			);
+
+			// Print the original error's message too, as sometimes the error isn't fixed by `pnpm install`.
+			console.error(
+				( chalk.grey || identity )( `The original error message was ${ error.message }` )
```

Comment:
This looks like very poor formatting when it happens in my testing:

```
The original error message was Cannot find package 'chalk' imported from /home/zinigor/workspace/jetpack/tools/cli/helpers/checkEnvironment.js
```
Can we quote the original message somehow? Even if we just put a colon and a newline on the end of our text?

Reply:
If we're going to print it before the prompt to `pnpm install`, we may as well just print it without elaboration.